projects/skia/BUILD.gn.diff

```diff
@@ -254,12 +254,9 @@ test_app("webp_encoder") {
 }
 
 test_app("skottie_json") {
-  sources = [
-    "fuzz/oss_fuzz/FuzzSkottieJSON.cpp",
```

Comment:
We still want this, no?

Reply:
It appears to still work without it.  Doesn't modules/skottie:fuzz bring this in?

Reply:
Yes, modules/skottie:fuzz should get you the source + needed defines.

Reply:
Oh totally missed that.  Thanks!